lahja/base.py

```diff
@@ -121,6 +121,25 @@ def is_connected_to(self, endpoint_name: str) -> bool:
         """
         pass
 
+    @abstractmethod
+    def broadcast_nowait(
+        self, item: BaseEvent, config: Optional[BroadcastConfig] = None
+    ) -> None:
+        """
+        A sync compatible version of :meth:`~lahja.base.EndpointAPI.broadcast`
+
+        .. warning::
+
+            Heavy use of :meth:`~lahja.base.EndpointAPI.broadcast_nowait` in
+            contiguous blocks of code without yielding to the `async`
+            implementation should be expected to cause problems.  Without
+            yielding, the braodcasted events will typically build back-pressure
```

Comment:
nitpick: the problem is that they'll fail to provide back-pressure! Maybe: "the broadcasted events will fail to provide backpressure and queue up until control is returned back to the event loop. This could result in unbounded memory growth or errors due to full queues"

Reply:
There's also a typo there: `braodcasted` -> `broadcasted`

Reply:
I decided to just remove the last sentence since each implementation could have it's own problems dependning on how it's implemented.